### ISSUE TITLE
Cypress was blocked by anti-virus software

### ISSUE THREAD

**Issue by JoeyLi-1** [cypress-io/cypress#21554]
### Current behavior

![image](https://user-images.githubusercontent.com/20162464/169217622-e4b4e987-637b-43de-a35b-f828a79f173d.png)

No matter npx open or npx run. It failed with same network issue.

### Desired behavior

It will work normally.

### Test code to reproduce

Sorry guys. I can`t give you our private code.

### Cypress Version

7.6.0

### Other

"Turning off the Network Threat Protection setting in Sophos antivirus solved the problem." This workaround works for us as well.

But!!! The problem is that our company`s policy does not allow us to turn off that setting (I doubt any company will allow that operation). So, please cypress team. Please fix this issue. Thank you!

**Comment by mjhenkes** [comment]
This is a duplicate of https://github.com/cypress-io/cypress/issues/14175
also see: https://community.sophos.com/intercept-x-endpoint/f/discussions/134136/sophos-network-threat-detection-is-blocking-cypress-automation-tool
